Automattic/wp-calypso
# Scroll the page to the RMT section when clicking back in details and update copy

Change the copy for some parts of RMT section.



Related to https://github.com/Automattic/dotcom-forge/issues/8141

## Proposed Changes

* Update the copy and add a new section in the details page of RMTs
* Scroll the page to RMTs section when the user clicks back

## Why are these changes being made?


*

## Testing Instructions



* Use the live link
* Notice the new text section in details
* Notice the copy changes
* Make sure that the back button scrolls the page to the RMT section

## Pre-merge Checklist
